README.md

```diff
@@ -2,6 +2,7 @@
 
 - 🐬 [Senior Blockchain Developer](/sr-dev.md)
 - 🐙 [Application Engineer](/ae.md)
+- 🦈 [Senior Product Developer - Data Farming](/df.md)
```

Comment:
"Product" isn't needed. And "Blockchain" helps attract actual blockchain developers, especially due to keyword searches.

Therefore I recommend changing the line to:
"Senior Blockchain Developer - Data Farming"